designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -172,152 +157,153 @@ class ManagementResources(
           }
         } ~
         path("deploy" / ProcessNameSegment) { processName =>
-          (post & processId(processName) & parameters(Symbol("savepointPath"))) { (processId, savepointPath) =>
-            canDeploy(processId) {
-              withDeploymentComment { deploymentComment =>
+          (post & processId(processName) & entity(as[Option[String]]) & parameters(Symbol("savepointPath"))) {
```

Comment:
I don't like this approach. Let's add a normal DTO with the `comment` and `savepointPath`. We occurred a limit of adding data on every possible way :laughing: The same regarding other places where the `comment` is in entity

Reply:
I agree with this suggestion, but I'd like to do it as a separate change. I'm a bit afraid that this change will turn out to be lengthy and it doesn't directly correlate to the deployment comment change. Would a separate PR be ok?

Reply:
Ahh LOL, I just checked that this change didn't change the API, this `Option[String]` already  was in the body. Excuse me for the confusion.  LGTM